sklearn/neighbors/tests/test_lof.py

```diff
@@ -99,8 +98,7 @@ def test_lof_values():
     assert_array_almost_equal(-clf2.score_samples([[1., 1.]]), [s_1])
 
 
-@ignore_warnings(category=DeprecationWarning)
-# contamination changed to 'auto' 0.22
+@pytest.mark.filterwarnings('ignore:default contamination parameter 0.1')
```

Comment:
Should we be giving the version number of scikit-learn where this line should be remove: "# 0.22" at the end of the line?